API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
+
+
+<pre>Skills</pre>
+name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+image_url | UrlField  
+image_retina_url | UrlField  
+
+<pre>WebLitSkills</pre>
+name | CharField  
+short_name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+**Skills** | *:*  
+**Topic** | 1:*  
+
+<pre>Topic</pre>
+name | CharField  
+description | TextField  
+
+<pre>Competencies</pre>
+text | TextField  
+
+<pre>LessonPlan</pre>
+title | CharField  
+description | TextField  
+short_description | TextField  
+learning_objectives | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+level | CharField (choices)  
+developed_by | TextField  
```

Comment:
The current data on the learning site suggests otherwise. There is no consistency between them. Some have multiple authors, some have one, in some only one author has a link, in others multiple authors have different links, some have "author - blah blah blah", etc.
Unless we are willing to modify the existing data, I think Ali's suggestion of having it be HTML is fine (we can validate it when users enter data but this shouldn't be reflected in the model)